jsk_robot_common/jsk_robot_startup/lifelog/common_logger.launch

```diff
@@ -310,6 +339,22 @@
       </rosparam>
     </node>
 
+    <!-- human pose logger -->
+    <node if="$(arg save_human_pose)"
+          name="human_pose_logger"
+          pkg="jsk_robot_startup" type="mongo_record.py"
+          machine="$(arg machine)"
+          respawn="$(arg respawn)">
+      <rosparam subst_value="true">
+        database: $(arg database)
+        topics:
+        - /edgetpu_human_pose_estimator/output/poses
```

Comment:
you can log all the messages like `~output/class`.
logging `~output/poses` only may not be enough.

https://github.com/jsk-ros-pkg/coral_usb_ros/blob/26cba3835b8b13d92738cb7ca3c76e7164f323a0/python/coral_usb/human_pose_estimator.py#L53-L59